src/components/AnchorForCommentsOnly/BaseAnchorForCommentsOnly/index.native.js

```diff
@@ -14,14 +20,28 @@ const BaseAnchorForCommentsOnly = ({
     shouldDownloadFile,
     ...props
 }) => (
-    <Text
-        style={StyleSheet.flatten(style)}
+    <PressableWithSecondaryInteraction
+        onSecondaryInteraction={
+            (event) => {
+                showContextMenu(
+                    contextMenuTypes.link,
+                    event,
+                    href,
+                    linkRef.current,
+                );
+            }
+        }
         onPress={() => (shouldDownloadFile ? fileDownload(href) : Linking.openURL(href))}
```

Comment:
I think, It should have an action on the contextMenu as well. `Open Link` which can follow what you are doing now.

Not sure if the click to open feature is yet planned. cc: @roryabraham.

Reply:
@roryabraham can you clarify @parasharrajat 's comment here? Doesn't seem like a bad idea to add Open Link to the context menu however in my testing a single/normal press opens the link. Not sure the exact scenario where they would get a download instead of going to the link.

Reply:
Ah, I missed this. Yeah, I don't think we really need `Open Link` in the context menu, but since you've already added it we can leave it.

Reply:
Let's ask in #expensify-open-source